src/Core/MySQL/PacketsPreparedStatements.cpp

```diff
@@ -0,0 +1,34 @@
+#include <Core/MySQL/PacketsPreparedStatements.h>
+#include <IO/WriteHelpers.h>
+#include <Common/logger_useful.h>
+
+namespace DB
+{
+namespace MySQLProtocol
+{
+    namespace PreparedStatements
+    {
+        size_t PrepareStatementResponseOK::getPayloadSize() const
+        {
+            return 13;
```

Comment:
I count 12 bytes (l. 18ff).

EDIT: Ah, with l. 24, it is 13 indeed. To make this more clear, maybe:

```cpp
static const auto resultset_metadata_none = '\n';
buffer.write(reinterpret_cast<const char *>(&resultset_metadata_none), 1);
```

Reply:
Btw., looking at the [docs](https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_com_stmt_prepare.html#sect_protocol_com_stmt_prepare_response_ok), the variable should actually be called `metadata_follows`.

Reply:
I think I can make it more clear by defining it as a sum with comments on what each portion of bytes represents.